app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,41 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = Food.all
+  end
+
+  def show
+    @food = Food.find(params[:id])
+    @user = User.find(params[:user_id])
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @food = @user.foods.new(food_params)
+
+    respond_to do |format|
+      format.html do
+        if @food.save
+          redirect_to user_foods_path(@user), notice: 'Food was successfully created.'
+        else
+          render :new
+        end
+      end
+    end
+  end
+
+  def destroy
+    @food = current_user.foods.find(params[:id])
```

Comment:
While you did an amazing job working on the project requirements You did not stick to the wireframe design provided for you. 

This is your design:

![Screenshot 2023-06-24 at 3 14 00 AM](https://github.com/obed300/recipe-app/assets/79506105/390b2d08-4ef5-4e32-acc1-3fada77b6f8d)
![Screenshot 2023-06-24 at 3 14 06 AM](https://github.com/obed300/recipe-app/assets/79506105/48250bd8-0154-477a-bb9e-1a90d680f9ef)
![Screenshot 2023-06-24 at 3 14 13 AM](https://github.com/obed300/recipe-app/assets/79506105/1a017de2-6db6-4be5-9ecd-d9fcebaa149b)
![Screenshot 2023-06-24 at 3 13 41 AM](https://github.com/obed300/recipe-app/assets/79506105/46c87a3b-6798-49dd-a8e3-7400906c0940)
![Screenshot 2023-06-24 at 3 13 52 AM](https://github.com/obed300/recipe-app/assets/79506105/1d8c03d4-2c3c-4eab-a341-806fa02149df)


Please stick to this wireframe design:

![Screenshot 2023-06-24 at 3 10 06 AM](https://github.com/obed300/recipe-app/assets/79506105/d2888f82-f557-45b0-bf4f-e7918d9097e4)
![Screenshot 2023-06-24 at 3 11 07 AM](https://github.com/obed300/recipe-app/assets/79506105/482819f5-ef9e-44d9-9507-5d41d027d35d)
![Screenshot 2023-06-24 at 3 10 28 AM](https://github.com/obed300/recipe-app/assets/79506105/66f3c6fc-5e93-4b16-9728-5fb1d7bd0d8c)
![Screenshot 2023-06-24 at 3 11 22 AM](https://github.com/obed300/recipe-app/assets/79506105/7d28bfa9-16b5-4d00-9b28-5a0f940b1cb0)
![Screenshot 2023-06-24 at 3 10 48 AM](https://github.com/obed300/recipe-app/assets/79506105/97db61f2-93d1-457c-bf69-985982fa7599)

If You are unsure please kindly follow these requirements to avoid making mistakes:

Design/Views


The application has a login and Registration page, built with the Devise gem.


Food List

Has an index page for food list that matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/food_index.png?raw=true).
Has a add new food item button that leads to the form that allows users to add new food.
Can delete a food item from the lists - only by owners



Recipies list:

Has an index page for recipes list that matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_index.png?raw=true).
You can remove a recipe item only if you're the owner
Clicking on a recipe item takes you to the recipe details page /recipes/:id/ as shown in the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true)

On the recipe show page, you should be able to add ingredient.
On the recipe show page, you should be able to generate a shopping list that matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/general_shopping_list.png?raw=true) and url /general_shopping_list




Public recipe list:

Has public recipes page listing all publicly available recipes from other users with their names and total prices. Page has to match the structure of the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/public_recipe_index.png?raw=true)

Clicking on a recipe item should lead to recipe details.



Recipe details

Recipe details page matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true)

Clicking on "Add ingredient" should lead to the form that allows user to add new food, if the user is the owner of the recipe.

Please Pay attention to detail following design rules is very important